app/controllers/admin/npq/applications/eligibility_status_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Admin
+  module NPQ
+    module Applications
+      class EligibilityStatusController < Admin::BaseController
+        StatusOption = Struct.new(:value, :label)
+
+        skip_after_action :verify_authorized
+        skip_after_action :verify_policy_scoped
+
+        def edit
+          @status_options = [
+            StatusOption.new(:marked_ineligible_by_policy, "Not eligible for funding"),
+            StatusOption.new(:awaiting_more_information, "Awaiting more information"),
+            StatusOption.new(:re_register, "Needs to re-register"),
+          ]
+
+          @npq_application = NPQApplication.find(params[:id])
+        end
+
+        def update
+          npq_application = NPQApplication.find(params[:id])
+          npq_application.assign_attributes(eligiblity_status_params)
+
+          if npq_application.save
+            name = npq_application.participant_identity.user.full_name
+
+            flash[:success] = {
+              title: "#{name} updated",
+              content: "#{name} has been marked '#{npq_application.funding_eligiblity_status_code.humanize.downcase}'",
```

Comment:
Not sure what you meant exactly as if i move into the transaltion file i will still have to create the translations in two places right? ie the edit and update unless i select the translation higher up so instead of `....npq.applications.eligibility_status.edit.x` it would be `....npq.applications.eligibility_status.x` I think what it says above makes sense?